admin/config.js

```diff
@@ -1,5 +1,7 @@
 'use strict'
 const os = require('os')
+const toBool = require('to-bool')
```

Comment:
👌 there are quite a few others that need this

Reply:
Done - other bools converted